helpers/helpers.go

```diff
@@ -198,10 +198,11 @@ func UnpackFile(file string, dest string) error {
 	return nil
 }
 
-// CopyFile is used during the build process to copy a given file to the target
-// instead of dealing with the particulars of hardlinking.
+// CopyFile copies a file, overwriting the destination if it exists. It is used
+// during the build process to copy a given file to the target instead of
+// dealing with the particulars of hardlinking.
```

Comment:
Optional: I'd cut the comment about where it is used.

Reply:
Yeah, almost did. Should have.

Reply:
Did.